docs/contributing/editing-docs.md

```diff
@@ -0,0 +1,47 @@
+
+The [Islandora Documentation Interest Group](https://github.com/islandora-interest-groups/Islandora-Documentation-Interest-Group) holds regular meetings to discuss how to improve and expand Islandora's documentation. If you are interested in becoming involved, all are welcome to join. If you just want to make a quick contribution to the existing documentation, that process is outlined here. 
+
+## Before you start
+
+To edit documentation in Islandora, you must:
+
+- have a GitHub account.
+- be a member of the Islandora [GitHub Organization](https://github.com/orgs/Islandora/people).
+    - This is only needed if you want to commit changes directly to the documentation. Non-members may still make pull requests. To request an invite, email community@islandora.ca or ask on the Islandora Slack. 
+- have a basic familiarity with markdown, or a good [cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+- follow the [Documentation Style Guide](docs_style_guide.md)
+
+## How to edit this documentation
+
+- Starting from the page you want to edit, look for the pencil icon in the top righthand corner of the page and click on it
+
+![Edit page button](../assets/editing-docs-button.png)
+
+- You will be taken to GitHub. If you are not logged in, you will be prompted to log in or register for an account. If you are logged in, you will be taken to an in-browser editor where you will view the page in markdown.
+
+![GitHub documentation GUI editor](../assets/editing-docs-gui.png)
+
+- Make your changes directly in the browser editor, then scroll to the bottom to "commit" your changes. This is the equivalent of saving.
+
+For the next step, you have two choices:
+
+- **Option 1: Small Changes**: If the change you made was very small, and wording and semantics are left untouched, such as a formatting or typo correction, you may push your changes into the documentation immediately by selecting _commit directly to the `main` branch_ and clicking **Commit changes**. 
+
+![commit directly to main branch](../assets/editing-docs-push.png)
+
+- **Option 2: Substantial Changes**: If the change you made is more substantial, and involves new wording, significant formatting changes, or other changes that you would prefer to have reviewed by someone else, create a new temporary branch and pull request by selecting _Create a new branch for this commit and start a pull request_. Give your branch a brief name starting with your username, with no spaces, indicating what it is for. Then click **Commit changes**.
```

Comment:
technical question - I don't think you'll be able to make a new branch if you're not part of the islandora github org (i.e. with write access to the repo)? 

Here's the results of trying to edit a file in islandora_ocr, which i don't have write access to: a note at the top of the page that they've created me a fork, and the button says "propose changes". 

![Screen Shot 2020-10-26 at 11 50 12 AM](https://user-images.githubusercontent.com/1943338/97187768-8c7aa500-1781-11eb-97ec-c46eb58e23f0.png)
![Screen Shot 2020-10-26 at 11 50 38 AM](https://user-images.githubusercontent.com/1943338/97187778-8e446880-1781-11eb-8f6f-dc2680aa7885.png)

Reply:
I hadn't considered that. I guess the choose your own adventure needs one more branch. I'll add that in.

Reply:
It's not so much a "choose your own adventure" - while we see it that way, someone new to this would not. 

Maybe assume they're "a general person" and talk about "proposing changes" first, yadda yadda github fork. 

If you're a member of the Islandora github organization, the options will be to 'create a branch and make a PR" or to merge directly. In general, do not merge directly into main. Create a PR. 

[image of "pull request" selected. Is there a way to make that the default?]

Pull Request Policy: (does it make sense to separate out the "review" from "merge" here?) PRs must be approved before they are merged. Any [????] can approve a PR. However, PRs can be merged only by someone who is not in the same organization as the author.  (the weird thing here is if it's at all collaborative, it would prevent people who are involved from being able to take action. like recently, i merged a PR of yours @manez that i'd had a commit in.)

(Only then, describe the exception)
PR versus Merge directly:  If your change is _extremely_ minor: a formatting or spelling correction that does not change the semantics of the document, you can skip the PR process and merge directly into the main branch. This option is available to those with write access, i.e. members of the islandora github org, use your power responsibly yadda yadda.

Reply:
I rewrote the "steps" the include what happens if you don't have write access and pushed the "exception" into a note block at the end. 

And added another clause under "Before you start" to link out to some of the GitHub terms we're throwing around.